src/components/App/App.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
Common Note: need to fix all console errors in the project

Reply:
I'm not sure I understood you